projects/demo/src/modules/components/input-phone-international/index.html

```diff
@@ -226,10 +221,18 @@
                     [tuiDropdownOffset]="dropdown.offset"
                     [tuiTextfieldCleaner]="textfield.cleaner"
                     [tuiTextfieldSize]="textfield.size"
-                    [(countryIsoCode)]="countryIsoCode"
                 >
-                    Type a phone number
-                </tui-input-phone-international>
+                    <label tuiLabel>Type a phone number</label>
+                    <input
+                        tuiInputPhoneInternational
+                        [countries]="countries"
+                        [countrySearch]="countrySearch"
+                        [formControl]="formControl"
+                        [invalid]="control.invalid"
+                        [readOnly]="control.readonly"
```

Comment:
https://github.com/user-attachments/assets/523952e6-e1bb-46d4-bd95-4776f9d70844

Reply:
Looks like we do not have readonly design for dark mode in new textfields also. Writing it here just so I don't forget to fix it